packages/boxplot/src/BoxPlotLegends.tsx

```diff
@@ -0,0 +1,22 @@
+import { BoxLegendSvg } from '@nivo/legends'
+import { BoxPlotLegendProps, LegendData } from './types'
+
+interface BoxPlotLegendsProps {
+    width: number
+    height: number
+    legends: [BoxPlotLegendProps, LegendData[]][]
+}
+
+export const BoxPlotLegends = ({ width, height, legends }: BoxPlotLegendsProps) => (
+    <>
+        {legends.map(([legend, data], i) => (
+            <BoxLegendSvg
+                key={i}
+                {...legend}
+                containerWidth={width}
+                containerHeight={height}
+                data={legend.data ?? data}
```

Comment:
Added a comment and a story about custom legends. 

Actually, this is a remnant from adapting code from @nivo/bar. I think this feature might be used there to support using the legend to toggle visibility of individiaul series. That interactivity is not implemented here. But this code could remain to allow users to create a completely custom legend. It would be an esoteric feature, but could be useful in complex layouts with multiple charts, e.g. chart A on the left, chart B on the right, and two legends in the right margin of chart B.